deploy/scripts/src/common/utils/jslib/index.d.ts

```diff
@@ -1 +1,2 @@
 export const uuidv4: () => string
+export const randomString: (length: number) => string
```

Comment:
```suggestion
export const randomString: (length: number, charset?: string) => string
```
Missing an optional parameter in the type definition

Reply:
Done